src/app/export/parse-data.ts

```diff
@@ -1,44 +1,41 @@
-import { Trial } from '../trialscope';
+import { ResearchStudySearchEntry, Facility } from '../services/search.service';
 
-const clearPunctuation = (entry: string) => entry.replace(/"/g, '');
-
-export const UnpackMatchResults = (result: Partial<Trial>[]) => {
+export const UnpackResearchStudyResults= (result: ResearchStudySearchEntry[]): object[] => {
```

Comment:
I think it would be useful to add a clarifying comment that this function is used for exporting saved trials to a csv.

Reply:
comment added